extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -192,11 +179,23 @@ export class AmpAdApiHandler {
    * @private
    */
   updateSize_(height, width, source, origin) {
-    this.baseInstance_.attemptChangeSize(height, width).then(() => {
-      this.sendEmbedSizeResponse_(
-        true /* success */, width, height, source, origin);
-    }, () => this.sendEmbedSizeResponse_(
-        false /* success */, width, height, source, origin));
+    let newHeight, newWidth;
```

Comment:
I agree. Or we can think about code sharing in more granular level, for example, share a util function like: `getContainerDimensionsForResize(containerElement, contentElement, newContentDimensions)` which returns `newContainerDimensions`.